PCbuild/pyproject-clangcl.props

```
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <__PyprojectClangCl_Props_Imported>true</__PyprojectClangCl_Props_Imported>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <_profrawFiles Include="$(OutDir)instrumented\$(TargetName)_*.profraw" />
+  </ItemGroup>
+
+  <Target Name="EnsureClangProfileData" BeforeTargets="PrepareForBuild"
+          Condition="'$(SupportPGO)' and $(Configuration) == 'PGUpdate'">
+    <Error Text="PGO run did not succeed (no $(TargetName)_*.profraw files) and there is no data to merge"
+           Condition="$(RequireProfileData) == 'true' and @(_profrawFiles) == ''" />
+  </Target>
+
+  <Target Name="MergeClangProfileData" BeforeTargets="PrepareForBuild"
+          Condition="'$(SupportPGO)' and $(Configuration) == 'PGUpdate'"
+          Inputs="@(_profrawFiles)"
+          Outputs="@(_profrawFiles->'$(OutDir)instrumented\profdata.profdata')">
+    <Exec
+      Command='"$(LLVMInstallDir)\bin\llvm-profdata.exe" merge -output=$(OutDir)instrumented\profdata.profdata $(OutDir)instrumented\*_*.profraw' />
+  </Target>
+
+  <Target Name="CleanClangProfileData" BeforeTargets="Clean">
+    <Delete Files="@(_profrawFiles->'$(OutDir)instrumented\%(Filename)%(Extension)')" TreatErrorsAsWarnings="true" />
```

```suggestion
    <Delete Files="@(_profrawFiles)" TreatErrorsAsWarnings="true" />
```

Unless the item group gets updated somewhere I didn't notice?

---

Ah - this works as well! Stumbled over that, too, and already simplified to `@(_profrawFiles->'%(FullPath)')`, but cannot beat your suggestion :)